documentation/frontend/README.md

```diff
@@ -0,0 +1,3 @@
+# Front-end Documentation
+
+Documentation for the Next.JS-based front-end can be found in this directory and the [frontend README.md](../../frontend/README.md).
```

Comment:
> …can be found in this directory…

The only thing in this directory really is the i18n docs. The instructions for running the frontend app live in the _other_ frontend README. At first, I was gonna suggest we didn't need a `/documentation/frontend/README` and a `/frontend/README` (`internationalization.md` could just be moved to the app directory). However, the api has a _lot_ more in the `documentation/` folder. We should be consistent, but I wonder if it might confuse new folx that some docs are w/ their specific apps in the monorepo and some are in the documentation folder? 

I'm not suggesting a change here. Just noting potential confusion.